apps/complaints/steps.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+module.exports = {
+	name: 'complaints',
+	steps: {
+		'/': {
+			template: 'step',
+			fields: ['name-text'],
+			next: '/step1'
+		},
+		'/step1': {
+			template: 'step',
+			fields: ['name-text']
+		}
+	}
+};
```

Comment:
See note above about steps and routes. This is actually a route with a `steps` field.